server/index.js

```diff
@@ -14,14 +14,8 @@ app.get('/api/locations', (req, res) => {
   }
   db.getOneProdLoc(productID)
     .then(data => res.send(data))
-    .catch(error => console.log(error))
+    .catch(error => res.sendState(404))
```

Comment:
@gigicupp is this supposed to be "sendState" or "sendStatus"

Reply:
Ah! yes. I will fix and merge.